src/components/BlogList/BlogList.jsx

```diff
@@ -3,10 +3,9 @@ import BlogCard from "../BlogCard/BlogCard";
 import "./BlogList.scss";
 
 const BlogList = ({ blogs }) => {
```

Comment:
Proptypes?

Reply:
We don't need to pass blogs as props as we're using redux store.

Reply:
We no longer passing blogs as props instead we are using redux store.